### ISSUE TITLE
error with "list" using example code

### ISSUE THREAD

**Issue by dracount** [facundoolano/app-store-scraper#104]
`store
  .list({
    collection: store.collection.TOP_FREE_IPAD,
    category: store.category.GAMES_ACTION,
    num: 2
  })
  .then(console.log)
  .catch(console.log);`

TypeError: Cannot read property 'href' of undefined
    at cleanApp (/sandbox/node_modules/app-store-scraper/lib/list.js:25:30)                                                                       0)
    at Array.map (<anonymous>)
    at /sandbox/node_modules/app-store-scraper/lib/list.js:48:17
    at process._tickCallback (internal/process/next_tick.js:68:7)

Not sure what I'm doing wrong here?

**Comment by facundoolano** [comment]
Node and app-store-scraper version?

**Comment by kblcuk** [comment]
This happens for me with latest `0.16` app-store-scraper version, Node 10.14.

The app.link (to which error message points to) seems to be an array nowadays:
```
[ { attributes:
     { rel: 'alternate',
       type: 'text/html',
       href: 'https://apps.apple.com/us/app/crazy-kick/id1469889140?uo=2' } },
  { 'im:duration': { label: '0' },
    attributes:
     { title: 'Preview',
       rel: 'enclosure',
       type: 'image/jpeg',
       href:
        'https://is4-ssl.mzstatic.com/image/thumb/Purple123/v4/02/25/26/022526db-b47f-2e9c-8867-5ef4eb25739f/pr_source.png/1024x1024bb.png',
       'im:assetType': 'preview' } } ]
```

**Comment by vadymus** [comment]
I have the same error testing the code. Node v10.14.1 and NPM 6.4.1. Would really like to see it fixed.

**Comment by kblcuk** [comment]
https://github.com/facundoolano/app-store-scraper/pull/105

**Comment by facundoolano** [comment]
Merged #105 
can @vadymus and @dracount confirm that solves the issue?

**Comment by vadymus** [comment]
It worked for me. Thanks!
